src/StandardReactiveSocket.cpp

```diff
@@ -213,13 +213,16 @@ void StandardReactiveSocket::createResponder(
   debugCheckCorrectExecutor();
 
   if (streamId != 0) {
+// TODO: Remove this: https://github.com/ReactiveSocket/reactivesocket-cpp/issues/243
+#ifndef DONT_VERIFY_STREAM_IDS
```

Comment:
As I understand it. We will allow server and client to generate the same Ids. In the lithium project we only create server streams and no client streams so they won't collide, but I hope we won't keep this hack in for a long time...